perun-rpc/src/main/java/cz/metacentrum/perun/rpc/methods/UsersManagerMethod.java

```diff
@@ -155,6 +155,46 @@ public Void call(ApiCaller ac, Deserializer parms) throws PerunException {
 		}
 	},
 
+	/*#
+	 * Set specific user type for specific user and set ownership of this user for the owner.
+	 *
+	 * @param specificUser int User <code>id</code>
+	 * @param specificUserType String specific user type
+	 * @param user int User <code>id</code>
```

Comment:
Probably should be `owner` instead of `user`.

Reply:
fixed